tools/clang/test/CodeGenSPIRV/vk.1p2.entry-point.hlsl

```diff
@@ -0,0 +1,40 @@
+// Run: %dxc -T ps_6_0 -E main -fspv-target-env=vulkan1.2
+
+// CHECK: OpEntryPoint Fragment %main "main"
+// CHECK-SAME: %_Globals %gSampler %gTex %MyCBuffer %gCBuffer %gSPInput %gRWBuffer %out_var_SV_Target
+
+          int           gScalar;   // 0
```

Comment:
Would you remove these comments? they don't make sense in this test anymore. (e.g. 0, Not included - 1, 1, etc)

Reply:
done